Formula/wtfutil.rb

```diff
@@ -0,0 +1,26 @@
+class Wtfutil < Formula
+  desc "The personal information dashboard for your terminal"
+  homepage "https://wtfutil.com"
+  url "https://github.com/wtfutil/wtf/archive/v0.10.2.tar.gz"
```

Comment:
@chenrui333 Can you also please bump this to 0.10.3?

Reply:
Definitely, doing it now.